INSTALL.rst.txt

```diff
@@ -0,0 +1,149 @@
+   http://scipy.org/scipylib/building/index.html
+
+
+Basic Installation
+==================
+
+To install numpy run::
+
+    python setup.py build -j 4 install --prefix $HOME/.local
+
+This will compile numpy on 4 CPUs and install it into the specified prefix.
+To perform an inplace build that can be run from the source folder run::
+
+    python setup.py build_ext --inplace -j 4
+
+The number of build jobs can also be specified via the environment variable
+NPY_NUM_BUILD_JOBS.
+
+
+Choosing compilers
+==================
+
+On OS X and Linux, all common compilers will work.  Note that for Fortran,
+``gfortran`` is strongly preferred over ``g77``, but if you happen to have both
+installed then ``g77`` will be detected and used first.  To explicitly select
+``gfortran`` in that case, do::
+
+    python setup.py build --fcompiler=gnu95
+
+Windows
+-------
+
+On Windows, building from source can be difficult.  Currently the most robust
+option is to use the Intel compilers, or alternatively MSVC (the same version
+as used to build Python itself) with Intel ifort.  Intel itself maintains a
+good `application note <https://software.intel.com/en-us/articles/numpyscipy-with-intel-mkl>`_
+on this.
+
+If you want to use a free compiler toolchain, the recommended compiler is MingwPy__.
+The older MinGW32 compiler set used to produce older .exe installers for Numpy
+itself is still available at https://github.com/numpy/numpy-vendor, but not
+recommended for use anymore.
+
+MingwPy__ http://mingwpy.github.io
+
+
+Building with optimized BLAS support
+====================================
+
+Configuring which BLAS/LAPACK is used if you have multiple libraries installed,
+or you have only one installed but in a non-standard location, is done via a
+``site.cfg`` file.  See the ``site.cfg.example`` shipped with Numpy for more
+details.
+
+Windows
+-------
+
+The Intel compilers work with Intel MKL, see the application note linked above. 
+MingwPy__ works with OpenBLAS.
+For an overview of the state of BLAS/LAPACK libraries on Windows, see 
+`here <http://mingwpy.github.io/blas_lapack.html>`_.
+
+OS X
+----
+
+OS X ships the Accelerate framework, which Numpy can build against without any
+manual configuration.  Other BLAS/LAPACK implementations (OpenBLAS, Intel MKL,
+ATLAS) will also work.
+
+Ubuntu/Debian
+-------------
+
+In order to build with optimized a BLAS providing development package must be installed.
+Options are for example:
+
+ - ``libblas-dev``: reference BLAS (not very optimized)
+ - ``libatlas-base-dev``: generic tuned ATLAS, it is recommended to tune it to
+   the available hardware, see /usr/share/doc/libatlas3-base/README.Debian for
+   instructions
+ - ``libopenblas-base``: fast and runtime detected so no tuning required but a
+   very recent version is needed (>=0.2.15 is recommended).  Older versions of
+   OpenBLAS suffered from correctness issues on some CPUs.
+
+The actual implementation can be exchanged also after installation via the
```

Comment:
"The package linked to when numpy is loaded can be chosen after installation via the"